.github/workflows/tests.yml

```diff
@@ -8,13 +11,59 @@ on:
   workflow_dispatch:
 
 jobs:
-  unittest-and-lint:
-    runs-on: ${{matrix.os}}
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Setup up Linter
+        run: |
+          pip install toml yapf
```

Comment:
You may want to specify the version of those dependencies

Reply:
Both toml and yapf seem to be pretty stable, and thus latest versions are fine from my point of view. Anyway, we explicitly specify the style in the pyproject.toml as proposed in #119

Reply:
I tend to agree with David: if tests fail due to updated tools, we should scramble to fix our code anyway. if this happens to be impossible, (I have trouble comming up with such a case), we can pin the versions then...